_posts/2023-3-13-collaborative-trailheads.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Trailheads, a Collaborative Approach"
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2023-03-13 12:00:00
+categories:
+  - Featured
+tags:
+  - data
+  - trailheads
+  - OSM
+  - OpenStreetMap
+---
+
+Certain datasets are best managed collectively. These datasets tend to be dynamic and have more than one steward. 
```

Comment:
```suggestion
At UGRC, we serve up datasets from a vast array of sources. Some, like municipal boundaries or voting precincts, are created in-house. Others are pulled from a single steward, like the land ownership data maintained by SITLA. A large part of our work is aggregating datasets from multiple stewards into coherent, state-wide datasets. Roads, parcels, and address points all fall into this category. 

However, there are some datasets that are so broad in their geographic scope and so dynamic in their content that even an aggregation strategy fails to keep up with their changes and fails to capture all the possible data sources.
```

Forgive my barging in, but I thought this might make for a stronger opening :)

Reply:
nice, thanks jake!

Reply:
i'm gonna commit it and i may put some of it in my own words.  good call.